src/ExpansionPanel/ExpansionPanel.js

```diff
@@ -0,0 +1,261 @@
+        expanded: nextProps.expanded,
+      });
+    }
+  }
+
+  isControlled = null;
+
+  onHeaderKeyUp({ keyCode }) {
+    if (keyCode === codes.enter || keyCode === codes.space) {
+      this.handleExpand();
+    }
+  }
+
+  handleFocus() {
+    this.setState({
+      focused: true,
+    });
+  }
+
+  handleBlur() {
+    this.setState({
+      focused: false,
+    });
+  }
+
+  handleExpand = () => {
+    const { disabled, onChange } = this.props;
+    if (disabled) {
+      return;
+    }
+    const expand = !this.state.expanded;
+    onChange(this, expand);
+    if (!this.isControlled) {
+      this.setState({ expanded: expand });
+    }
+  };
+
+  renderHeader() {
+    const ComponentProp = 'div';
+    const { classes, disabled, expandIcon, headerTitle, headerTitleProps } = this.props;
+    const { expanded, focused } = this.state;
+    return (
+      <ComponentProp
+        className={classNames(classes.header, {
+          [classes.headerHover]: !disabled,
+          [classes.headerExpanded]: expanded,
+          [classes.headerFocused]: focused,
+        })}
+        onClick={this.handleExpand}
+      >
+        <Typography
+          type="body2"
+          component="span"
+          className={classNames(classes.headerTitle, {
+            [classes.headerDisabled]: disabled,
+          })}
+          {...headerTitleProps}
+        >
+          {headerTitle}
+        </Typography>
+        {expandIcon &&
+          <IconButton
+            disabled={disabled}
+            className={classNames(classes.expandButton, {
+              [classes.expandButtonOpen]: expanded,
+            })}
+            component="div"
+            tabIndex={disableTabbing}
+            onClick={this.handleExpand}
+          >
+            {expandIcon}
+          </IconButton>}
+      </ComponentProp>
+    );
+  }
+
+  renderKeyboardFocusHolder() {
+    const ComponentProp = 'div';
+    const { classes, disabled } = this.props;
+    return (
+      <ComponentProp
+        className={classes.focusHolder}
+        tabIndex={!disabled && enableTabbing}
+        onKeyUp={e => this.onHeaderKeyUp(e)}
+        onFocus={() => this.handleFocus()}
+        onBlur={() => this.handleBlur()}
+      />
+    );
+  }
+
+  render() {
+    const { children, classes, className, disabled, unmountOnExit } = this.props;
+    const { expanded } = this.state;
+    return (
+      <Card
+        className={classNames(className, classes.root, {
+          [classes.disabled]: disabled,
+          [classes.expanded]: expanded,
+        })}
+      >
+        {this.renderHeader()}
+        {this.renderKeyboardFocusHolder()}
+        <Collapse in={expanded} transitionDuration="auto" unmountOnExit={unmountOnExit}>
+          {children}
+        </Collapse>
+      </Card>
+    );
+  }
+}
+
+ExpansionPanel.propTypes = {
+  /**
+   * The content of the expansion panel.
+   */
+  children: PropTypes.element,
+  /**
+   * Useful to extend the style applied to components.
```

Comment:
> is there any simple way to do that now inside the code-comment

Good question! The generated docs are markdown, so I had assumed that a markdown link `[anchor](path)` would just work, but you might have to try it and see.